src/tests/integration/kllvm/test_convert.py

```diff
@@ -46,20 +51,59 @@
 )
 
 
-@pytest.mark.parametrize('test_id,kore_text', TEST_DATA, ids=[test_id for test_id, *_ in TEST_DATA])
-def test_kore_to_llvm(test_id: str, kore_text: str) -> None:
+def _in_module(kore_text: str) -> str:
+    return f'[] module FOO {kore_text} endmodule []'
+
+
+DEF_TEST_DATA: Final = (
+    ('empty-modules', '[DefAttr{}("foo")] module FOO endmodule [ModAttr{}("bar")]'),
+    ('import', '[] module FOO endmodule [] module BAR import FOO [ImportAttr{}()] endmodule []'),
+    ('sort-decl', _in_module('sort Foo{S,T} [Attr{}()]')),
+    ('hooked-sort-decl', _in_module('hooked-sort Foo{S,T} [Attr{}()]')),
+    ('symbol-decl', _in_module('symbol Foo{S,T}(SortBar{S,T}, SortBaz{}) : SortInt [SymbolAttr{}()]')),
+    ('hooked-symbol-decl', _in_module('hooked-symbol Foo{S,T}(SortBar{S,T}, SortBaz{}) : SortInt [SymbolAttr{}()]')),
+    (
+        'alias-decl',
+        _in_module('alias Foo{S,T}(S) : T where Foo{S,T}(X : S) := X : T [AliasAttr{}()]'),
+    ),
+    ('axiom', _in_module(r'axiom {S} \rewrites{S}(X : S, \dv{SortInt{}}("1")) [AxiomAttr{}()]')),
+    ('claim', _in_module(r'claim {S} \rewrites{S}(X : S, \dv{SortInt{}}("1")) [AxiomAttr{}()]')),
+)
+
+
+@pytest.mark.parametrize('test_id,kore_text', PAT_TEST_DATA, ids=[test_id for test_id, *_ in PAT_TEST_DATA])
+def test_kore_pattern_to_llvm(test_id: str, kore_text: str) -> None:
```

Comment:
If you agree with the naming changes suggested above, then:

```suggestion
def test_pattern_to_llvm(test_id: str, kore_text: str) -> None:
```

(and similarly for the rest)